cmd/livepeer-log/livepeer-log.go

```diff
@@ -143,15 +143,27 @@ func main() {
 			panic(err)
 		}
 
+		print := func(text string) {
+			fmt.Fprintf(os.Stderr, "INFO|%s|%d|||%s\n", procname, mypid, text)
+		}
+
 		// Actually print with lots of lines!
-		for _, pipe := range []io.ReadCloser{stderr, stdout} {
-			go func(pipe io.ReadCloser) {
-				scanner := bufio.NewScanner(pipe)
-				for scanner.Scan() {
-					text := scanner.Text()
-					fmt.Fprintf(os.Stderr, "INFO|%s|%d|||%s\n", procname, mypid, text)
+		for i, pipe := range []io.ReadCloser{stdout, stderr} {
```

Comment:
Is it worth enforcing some sort of limit still, for when we accidentally dump a whole video file of binary data into the logs?

Reply:
I dunno, feels like the wrong level to handle that kind of thing, considering this is a script that we'd like to get rid of completely after moving to Mist's first-class external service thing. I'd want Loki or maybe containerd to enforce some kind of rate limit cap probably though.